Documentation/Functions/Select.rst

```diff
@@ -154,7 +154,18 @@ pidInList
       Allows to disable the :sql:`pid` constraint completely. Requirements: 
       :ts:`uidInList` *must* be set or the table *must* have the prefix 
       "static\_\*".
+      
+   .. note::
+      Check the doktype of your backend page. If you are trying to fetch records from
+      e.g. a folder, the :php:`$cObj->checkPid_badDoktypeList` method will insert the
+      following SQL into your query:
 
+      .. code-block:: sql
+
+      [...]WHERE (`your_requested_table_name`.`uid` = 0) AND [...]
+
+      Which will always result in an empty query result!
```

Comment:
Please try formulating this without exclamation mark...

Reply:
I rephrased it, you're right. It looks odd :)